prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-overview.component.ts

```diff
@@ -73,6 +73,47 @@ import { RegulatoryForm } from './regulatory-form.model';
         None
       </app-enrollee-property>
     </app-page-section>
+
+    <app-page-section>
+
+      <app-page-subheader>
+        <ng-container appPageSubheaderTitle>Other College Licence Information</ng-container>
+
+        <button *ngIf="showEditRedirect"
+                mat-icon-button
+                matTooltip="Edit College Licences"
+                (click)="onRoute(PaperEnrolmentRoutes.REGULATORY)">
+          <mat-icon>edit</mat-icon>
+        </button>
+      </app-page-subheader>
+
+
+      <ng-container *ngIf="!regulatory?.unlistedCertifications?.length; else unlistedCerts">
+        No unlisted certifications entered.
```

Comment:
Can you please check with someone from the Business whether this text makes sense to them?

Reply:
Text received from business and updated